src/components/SearchBar.tsx

```diff
@@ -95,7 +95,8 @@ export interface SearchBarCssClasses extends AutocompleteResultCssClasses {
 export type RenderEntityPreviews = (
   autocompleteLoading: boolean,
   verticalKeyToResults: Record<string, VerticalResultsData>,
-  onSubmit: (value: string, _index: number, itemData?: FocusedItemData) => void
+  handleSubmit: (value: string, _index: number, itemData?: FocusedItemData) => void,
+  returnEntityAria: (value: string) => string
```

Comment:
I might call this `getEntityAria` or `getAriaLabel` since I'm not sure what an entity aria is